misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -164,6 +167,13 @@ class JettyService @Inject internal constructor(
     server.stopTimeout = 25_000
     ServerConnectionStatistics.addToAllConnectors(server)
 
+    if (webConfig.gzip) {
+      gzipHandler.server = server
+      gzipHandler.minGzipSize = webConfig.minGzipSize
+      gzipHandler.addIncludedMethods("POST")
```

Comment:
what does this do?

Reply:
By default the gzip handler only runs on GET requests. This tells it to also run on POSTs